api/app/src/mappings/garmin/body-composition.ts

```diff
@@ -0,0 +1,81 @@
+import { Body } from "@metriport/api";
+import { groupBy } from "lodash";
+import { z } from "zod";
+import {
+  DataType,
+  garminMetaSchema,
+  garminTypes as t,
```

Comment:
Why not just `garminTypes`?

Reading this review I didn't know what `t` was in `garminBodyCompositionSchema` and had to scroll up to figure it out. Suggest making objects descriptive, with `z` being an exception as that's consistent wherever we use `zod`.

Reply:
I understand the initial context setup, but this could be a definition for people working on this project, which makes it simpler to write code. Surely, variables should convey meaning through naming, but just like we recognize `z` as `zod`, we could recognize `t` as the provider's types.

Reply:
Changed the code to use `garminTypes`